app/assets/stylesheets/responsive/_request_layout.scss

```diff
@@ -270,6 +270,8 @@ a.track-request-action {
     align-items: stretch;
   }
 
+  float: none;
```

Comment:
Without this the eu promo overlaps the action menu. I expect this is coming from either the `grid-column` or the `flexbox`, but I don't really understand the consequences of not having this floating.

![screen shot 2017-01-11 at 13 31 19](https://cloud.githubusercontent.com/assets/282788/21850393/4ed248ce-d802-11e6-8a05-900681fd7c80.png)